frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -45,7 +45,6 @@ export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
     },
     [props.nameSpace],
   );
-  const netBookYear = getIn(formikProps.values, withNameSpace('financials.0.netbook.fiscalYear'));
```

Comment:
Whilst this technically works since all 10 years of history from the current year are created, and in sorted order, and currently only contain netbook values, this doesn't work if another type of fiscal is added. Switching to searching based on the programmatically determined fiscal index is safer.

Reply:
it took my brain a minute to read "programmatically" lol